src/tests/ftest/dfuse/pil4dfs_dcache.yaml

```diff
@@ -0,0 +1,38 @@
+hosts:
+  test_servers: 1
+  test_clients: 1
+
+timeout: 200
+
+server_config:
+  name: daos_server
+  engines_per_host: 2
+  engines:
+    0:
+      pinned_numa_node: 0
+      fabric_iface: ib0
+      fabric_iface_port: 31317
+      log_file: daos_server0.log
+      storage:
+        0:
+          class: dcpm
+          scm_list: ["/dev/pmem0"]
+          scm_mount: /mnt/daos0
+    1:
+      pinned_numa_node: 1
+      fabric_iface: ib1
+      fabric_iface_port: 31417
+      log_file: daos_server1.log
+      storage:
+        0:
+          class: dcpm
+          scm_list: ["/dev/pmem1"]
+          scm_mount: /mnt/daos1
```

Comment:
Could we use `storage: auto` here?

Reply:
- [ ] use auto storage

Reply:
Fixed with commit eace57b3b479abd420dec07bf85cc46b4a1c9c31